web/src/locales/schema.json

```diff
@@ -52,16 +52,19 @@
 		"containers.create.success": {
 			"type": "string"
 		},
-		"containers.search": {
+		"location.search": {
 			"type": "string"
 		},
 		"containers.category": {
 			"type": "string"
 		},
-		"containers.location": {
+		"location": {
 			"type": "string"
 		},
-		"containers.location.placeholder": {
+		"location.placeholder": {
```

Comment:
We could group all the keys that contain the `location.` prefix and place them near `location.unknownWay`.
What do you think?